packages/account/src/Components/trading-assessment/trading-assessment-form.tsx

```diff
@@ -19,15 +32,16 @@ const TradingAssessmentForm = ({
     should_move_to_next,
     setSubSectionIndex,
     is_independent_section,
-}) => {
+    is_mobile,
```

Comment:
cant we get this from usestore instead of prop passing

Reply:
Since this not the main component which is being rendered , the store on the main component does not recognize the use store values from here and throws an here. So in order for store provider to recognize the useStore usage, passed it as a prop from the main component